grpc-server/src/main/scala/com/wavesplatform/api/grpc/AssetsApiGrpcImpl.scala

```diff
@@ -35,7 +35,7 @@ class AssetsApiGrpcImpl(assetsApi: CommonAssetsApi, accountsApi: CommonAccountsA
           .concatMapIterable(_.map { case (a, d) =>
             NFTResponse(a.id.toByteString, Some(assetInfoResponse(d)))
           })
-          .take(request.limit)
+          .takeLast(request.limit)
```

Comment:
This does not seem right: it would always return the last `request.limit` entries.

Reply:
Done